src/guide/forms.md

```diff
@@ -272,56 +272,56 @@ The `true-value` and `false-value` attributes don't affect the input's `value` a
   <option :value="{ number: 123 }">123</option>
 </select>
 ```
 
 ```js
-// when selected:
+//  quando está assinalado:
 typeof vm.selected // => 'object'
 vm.selected.number // => 123
 ```
 
-## Modifiers
+## Modificadores
 
 ### `.lazy`
 
-By default, `v-model` syncs the input with the data after each `input` event (with the exception of IME composition as [stated above](#vmodel-ime-tip)). You can add the `lazy` modifier to instead sync after `change` events:
+Por padrão, `v-model` sincroniza o elemento _input_ com os dados após cada evento `input` (exceto para o caso de composição IME [descrito anteriormente](#vmodel-ime-tip)). Você pode adicionar o modificador `lazy` para sincronizar depois do evento `change`:
 
 ```html
-<!-- synced after "change" instead of "input" -->
+<!-- sincronizado depois do "change" ao invés de "input" -->
 <input v-model.lazy="msg" />
 ```
 
 ### `.number`
 
-If you want user input to be automatically typecast as a number, you can add the `number` modifier to your `v-model` managed inputs:
+Se você deseja que a entrada do usuário seja automaticamente tipificada como um Number, pode adicionar o modificador `number` ao `v-model` do elemento:
 
 ```html
 <input v-model.number="age" type="number" />
 ```
 
-This is often useful, because even with `type="number"`, the value of HTML input elements always returns a string. If the value cannot be parsed with `parseFloat()`, then the original value is returned.
+Isto geralmente é útil, porque mesmo com `type="number"`, o valor do elemento HTML _input_ sempre retorna uma string. Se o valor não puder ser convertido com `parseFloat()`, então o valor original é retornado.
 
 ### `.trim`
 
-If you want whitespace from user input to be trimmed automatically, you can add the `trim` modifier to your `v-model`-managed inputs:
+Se você quiser que os espaços em branco do _input_ do usuário sejam automaticamente cortados, pode adicionar o modificador `trim` ao `v-model` do elemento:
 
 ```html
 <input v-model.trim="msg" />
 ```
 
-## `v-model` with Components
+## `v-model` com Componentes
 
-> If you're not yet familiar with Vue's components, you can skip this for now.
+> Se você ainda não está familiarizado com os componentes Vue, pode pular isto por enquanto.
 
-HTML's built-in input types won't always meet your needs. Fortunately, Vue components allow you to build reusable inputs with completely customized behavior. These inputs even work with `v-model`! To learn more, read about [custom inputs](./component-basics.html#using-v-model-on-components) in the Components guide.
+os tipos de _input_ nativos do HTML nem sempre atendem as suas necessidades. Felizmente, os componentes Vue te permitem construir _inputs_ reutilizáveis com comportamento completamente costumizável. Estes componentes também funcionam com `v-model`! Para saber mais, leia sobre [inputs customizados](./component-basics.html#using-v-model-on-components) no guia de Componentes.
```

Comment:
O parágrafo está começando com letra minúscula ("os tipos"...). Por gentileza, poderia corrigir?

Se possível, também tornar `Para saber mais, leia sobre [inputs customizados]` em `Para saber mais, leia sobre [_inputs_ customizados]` (já que, em palavras ou termos estrangeiros, devemos colocar em itálico).